### PR TITLE
Notification improvements

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

**Comment by apmasell** [comment]
This was written by @djcooke; I just extracted it. Therefore, I am reviewing it and giving it :+1:.

File: run-tools/src/main/java/uk/ac/bbsrc/tgac/miso/tools/run/RunFolderConstants.java [hunk 1]
```diff
@@ -49,7 +49,7 @@
   String LS454_SIGNAL_FOLDER_REGEX = "D_\\d{4}_\\d{2}_\\d{2}_\\d{2}_\\d{2}_\\d{2}_[A-z0-9\\+\\-]+_[signalProcessing|fullProcessingAmplicons].*";
   String LS454_IMAGE_FOLDER_REGEX = "D_\\d{4}_\\d{2}_\\d{2}_\\d{2}_\\d{2}_\\d{2}_[A-z0-9\\+\\-]+_imageProcessingOnly.*";
 
-  String PACBIO_FOLDER_REGEX = ".*/\\d{4}_\\d{2}_\\d{2}_.*";
-  String PACBIO_FOLDER_CAPTURE_REGEX = ".*/(\\d{4}_\\d{2}_\\d{2})_(.*)/.*";
-  String PACBIO_FOLDER_NAME_GROUP_CAPTURE_REGEX = "(\\d{4}_\\d{2}_\\d{2})_(.*)";
+  String PACBIO_FOLDER_REGEX = ".*/\\d{8}_.*";
+  String PACBIO_FOLDER_CAPTURE_REGEX = ".*/(\\d{8})_(.*)/.*";
+  String PACBIO_FOLDER_NAME_GROUP_CAPTURE_REGEX = "(\\d{8})_(.*)";
 }
```
Comment (callunity) [inline run-tools/src/main/java/uk/ac/bbsrc/tgac/miso/tools/run/RunFolderConstants.java]: If we're merging into TGAC should we maintain the original regex as well?

Comment (djcooke) [inline run-tools/src/main/java/uk/ac/bbsrc/tgac/miso/tools/run/RunFolderConstants.java]: Yes, this regex is specific to OICR. We should probably make this configurable somewhere

**Comment by djcooke** [comment]
I'm getting all kinds of exceptions when trying to post runs from the Notification Server. I really don't think it's caused by anything here, but I'll investigate further

**Comment by djcooke** [comment]
I removed the RunFolderConstants changes. They can stay in the OICR branch for now and I'll make a ticket to make them configurable. The errors I received upon receiving notifications from the Notification Server are not caused by anything in this pull request, and I will make another separate ticket to fix them

**Comment by apmasell** [comment]
I've add the RunFolder changes back to the oicr_redux branch.

**Comment by djcooke** [comment]
Just noticed the added tests aren't included in a test suite, so they don't get run automatically. And... some of them are failing. Will fix

**Comment by djcooke** [comment]
Tests now run and pass. This is ready to merge for real after someone else reviews

**Comment by frankie13** [comment]
This looks good to me - I have also tested against our prod database so :+1:

**Pull request merged**